app/models/venue.rb

```diff
@@ -5,4 +5,5 @@ class Venue < ApplicationRecord
   has_many :accessibility_features, through: :venue_features
   geocoded_by :address
   after_validation :geocode, if: :will_save_change_to_address?
+  has_one_attached :photo
```

Comment:
I think we want has_many_attached as it will still allow only one but would mean multiple photos of different features can be included?

Reply:
Done